visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts

```diff
@@ -62,51 +62,49 @@ export class CodeMapLabelService implements CameraChangeSubscriber {
 		const labelYOrigin = y + node.height
 		const labelZ = (z + node.length / 2) * multiplier.z
 
-		if (node.attributes?.[state.dynamicSettings.heightMetric]) {
-			let labelText = ""
+		let labelText = ""
 
-			if (options.showNodeName) {
-				labelText = `${node.name}`
-			}
-			if (options.showNodeMetric) {
-				if (labelText !== "") {
-					labelText += "\n"
-				}
-				labelText += `${node.attributes[state.dynamicSettings.heightMetric]} ${state.dynamicSettings.heightMetric}`
+		if (options.showNodeName) {
+			labelText = `${node.name}`
+		}
+		if (options.showNodeMetric) {
+			if (labelText !== "") {
+				labelText += "\n"
 			}
+			labelText += `${node.attributes[state.dynamicSettings.heightMetric]} ${state.dynamicSettings.heightMetric}`
```

Comment:
Are we safe that `node.attributes` is indeed an object? We originally had a safeguard at the top (`node.attributes?.`).